universal-login-relayer/test/core/services/EncryptedWalletHandler.test.ts

```diff
@@ -5,37 +5,46 @@ import {EncryptedWalletHandler} from '../../../src/core/services/EncryptedWallet
 import {TEST_ENCRYPTED_WALLET_JSON, StoredEncryptedWallet} from '@unilogin/commons';
 import {expect} from 'chai';
 import {createTestEmailConfirmation} from '../../testhelpers/createTestEmailConfirmation';
+import {EmailConfirmationValidator} from '../../../src/core/services/validators/EmailConfirmationValidator';
 
 describe('INT: EncryptedWalletHandler', () => {
-  const notConfirmedEmail = 'notConfirmed@email.com';
-  const confirmedEmail = 'confirmed@email.com';
+  const confirmedMail = 'confirmed@email.com';
```

Comment:
Why did you rename confirmedEmail  => confirmedMail?

Reply:
Done